test/unit/spec/PollController_spec.js

```diff
@@ -77,42 +77,32 @@ function (Okta, Q, OktaAuth, Util, PollingForm, Expect, Router, $sandbox, resPol
             });
           });
       });
-    });
-  });
-
-  Expect.describe('Polling', function () {
-    describe('API', function () {
       itp('starts polling on load', function () {
-        return setup({}, [resPolling, resPolling, resPolling, resSuccess]).then(function (test) {
-          return Expect.wait(function () {
-            return $.ajax.calls.count() > 3;
-          }, test);
-        })
-          .then(function () {
-            // first call is for refresh-auth
-            expect($.ajax).toHaveBeenCalledTimes(4);
-            // 1st poll
-            Expect.isJsonPost($.ajax.calls.argsFor(1), {
-              url: 'https://example.okta.com/api/v1/authn/factors/okta-poll/poll',
-              data: {
-                stateToken: '00_J1qxqyLs-6ZutUUWfbqm-1nqnW6n2o5z2wnBRHs'
-              }
-            });
-            // 2nd poll
-            Expect.isJsonPost($.ajax.calls.argsFor(2), {
-              url: 'https://example.okta.com/api/v1/authn/factors/okta-poll/poll',
-              data: {
-                stateToken: '00_J1qxqyLs-6ZutUUWfbqm-1nqnW6n2o5z2wnBRHs'
-              }
-            });
-            // 3rd poll
-            Expect.isJsonPost($.ajax.calls.argsFor(3), {
-              url: 'https://example.okta.com/api/v1/authn/factors/okta-poll/poll',
-              data: {
-                stateToken: '00_J1qxqyLs-6ZutUUWfbqm-1nqnW6n2o5z2wnBRHs'
-              }
-            });
+        return setup({}, [resPolling, resPolling, resSuccess]).then(function () {
+          jasmine.clock().uninstall();
+          var originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+          jasmine.DEFAULT_TIMEOUT_INTERVAL = 2000;
```

Comment:
I dont think we need this. 
Just this should work right.
 jasmine.clock().install();
 jasmine.clock().tick(5000);
expect($.ajax.calls.count()).toBe(3)

Reply:
@magizh-okta I saw a similar usage in https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/PrimaryAuth_spec.js#L765, don't think that should matter, but I have tried removing that bit and it still seems to fail when all tests run together but passes when this spec is run individually.

Reply:
Reverting back to my old change